### PR TITLE
outlook mail: tell the LLM to print links

### PR DESCRIPTION
**Pull request by g-linville**
for https://github.com/gptscript-ai/desktop/issues/361

### CODE REVIEW

**Review by thedadams — APPROVED** [review]

File: apis/outlook/mail/manage/tool.gpt [hunk 1]
```diff
@@ -9,4 +9,4 @@ Type: context
 
 You have access to a collection of operations in the Microsoft Outlook Mail API.
 Do not output mail folder IDs or message IDs because they are not helpful for the user. The message IDs are needed for getting message details, deleting a message, or moving a message.
-When printing a list of messages for the user, include the body preview. When printing a single message and its details, print the full body.
+When printing a list of messages for the user, include the body preview. When printing a single message and its details, print the full body. Always include the email link.
```
Comment (StrongMonkey) [inline apis/outlook/mail/manage/tool.gpt:12]: Not sure if telling it to print link with markdown format will yield better visual result

**Review by StrongMonkey — APPROVED** [review]

File: apis/outlook/mail/manage/tool.gpt (hunk 1 above)
Comment (tylerslaton) [reply apis/outlook/mail/manage/tool.gpt:12]: I think we should do this just to be explicit. LLMs are pretty good at writing markdown.

**Review by tylerslaton — APPROVED** [review]
Approving since this will work fine as is. Being a bit more explicit about the format is something I'd also advocate for but is not blocking for me.

**Comment by g-linville** [comment]
I always see gpt-4o use Markdown, so I think I'm just gonna merge this as is.

**Pull request merged**